ahiv/kafka/util.h

```diff
@@ -1,16 +1,25 @@
 // Copyright 2019 Ahiv Authors. All rights reserved. Use of this source  code
 // is governed by a MIT-style license that can be found in the LICENSE file.
 
-#ifndef AHIV_KAFKA_UTIL_H
-#define AHIV_KAFKA_UTIL_H
+#ifndef AHIV_KAFKA_UTIL_H_
+#define AHIV_KAFKA_UTIL_H_
 
 namespace ahiv::kafka {
-    static void DumpAsHex(const char* data, std::size_t length) {
-        std::cout << length << std::endl << std::flush;
-        for (int j = 0; j < length; j++) printf("%02X ", data[j]);
-        std::cout << std::flush;
-    }
+
+// DumpAsHex prints the input to stdout as a hex string.
+static void DumpAsHex(const char* input, std::size_t length) {
+  DumpAsHex(input, length, output);
```

Comment:
Where does output come from?

Reply:
Oh yeah there is one semantical error I anticipated. I'll change it to `std::out` real quick.